tests/test_fixity.py

```diff
@@ -0,0 +1,472 @@
+                "json.return_value": {},
+            },
+            spec=requests.Response,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {
+                    "success": True,
+                    "message": "",
+                    "failures": {
+                        "files": {"missing": [], "changed": [], "untracked": []}
+                    },
+                    "timestamp": None,
+                },
+            },
+            spec=requests.Response,
+        ),
+    ],
+)
+def test_scan(_get):
+    aip_id = uuid.uuid4()
+
+    response = fixity.scan(
+        aip=str(aip_id),
+        ss_url=STORAGE_SERVICE_URL,
+        ss_user=STORAGE_SERVICE_USER,
+        ss_key=STORAGE_SERVICE_KEY,
+        session=SESSION,
+    )
+
+    assert response is True
+
+    assert _get.mock_calls == [
+        mock.call(
+            f"{STORAGE_SERVICE_URL}api/v2/file/{aip_id}/",
+            params={"username": STORAGE_SERVICE_USER, "api_key": STORAGE_SERVICE_KEY},
+        ),
+        mock.call(
+            f"{STORAGE_SERVICE_URL}api/v2/file/{aip_id}/check_fixity/",
+            params={"username": STORAGE_SERVICE_USER, "api_key": STORAGE_SERVICE_KEY},
+        ),
+    ]
+
+
+@mock.patch(
+    "requests.get",
+    side_effect=[
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {},
+            },
+            spec=requests.Response,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {
+                    "success": True,
+                    "message": "",
+                    "failures": {
+                        "files": {"missing": [], "changed": [], "untracked": []}
+                    },
+                    "timestamp": None,
+                },
+            },
+            spec=requests.Response,
+        ),
+    ],
+)
+@mock.patch(
+    "requests.post",
+    side_effect=[
+        mock.Mock(status_code=201, spec=requests.Response),
+        mock.Mock(status_code=201, spec=requests.Response),
+    ],
+)
+def test_scan_if_report_url_exist(_post: mock.Mock, _get: mock.Mock):
+    aip_id = uuid.uuid4()
+
+    response = fixity.scan(
+        aip=str(aip_id),
+        ss_url="",
+        ss_user=STORAGE_SERVICE_USER,
+        ss_key=STORAGE_SERVICE_KEY,
+        session=SESSION,
+        report_url=STORAGE_SERVICE_URL,
+    )
+
+    assert response is True
+
+    start_time = int(calendar.timegm(datetime.utcnow().utctimetuple()))
+    assert _get.mock_calls == [
+        mock.call(
+            f"api/v2/file/{aip_id}/",
+            params={"username": STORAGE_SERVICE_USER, "api_key": STORAGE_SERVICE_KEY},
+        ),
+        mock.call(
+            f"api/v2/file/{aip_id}/check_fixity/",
+            params={"username": STORAGE_SERVICE_USER, "api_key": STORAGE_SERVICE_KEY},
+        ),
+    ]
+    assert _post.call_count == 2
```

Comment:
You haven't added assertions against `.call_count` anywhere else which looks odd. But I'd consider the assertion unnecessary anyway since in the next line you're asserting against the actual calls to the mock which is more clear in my opinion.